README.md

```diff
@@ -32,6 +32,7 @@ OpenWhisk is a cloud-first distributed event-based programming service. It provi
 * [Native development](#native-development) (Mac and Ubuntu)
 * [Kubernetes](#kubernetes-setup)
 * [Vagrant](#vagrant-setup)
+* [Lean](#lean-setup)
```

Comment:
I would not put the Lean configuration into this top level list of deployment options.